_codux/boards/services-section.board.tsx

```diff
@@ -0,0 +1,74 @@
+            <div className={styles.content}>
+                <h2 className={styles.servicesTitle}> Write a Title Here</h2>
+                <p className={styles.servicesDescription}>
+                    This is the space to introduce visitors to the business or brand. Briefly
+                    explain who&apos;s behind it, what it does and what makes it unique. Share its
+                    core values and what this site has to offer.
+                </p>
+                <button className={commonStylesStyles.secondaryButtonWidth}>Explore</button>
+            </div>
+            <div className={styles.allCards}>
+                <div className={styles.card}>
+                    <div className={styles.iconContainer}>
+                        <img src={AnchorSvg} alt="" className={styles.icon} />
+                    </div>
+                    <h6 className={styles.serviceCardTitle}>Service Title</h6>
+                    <p className={styles.serviceCardDescription}>
+                        This is the space to describe the service and explain how customers or
+                        clients can benefit from it. It’s an opportunity to add a short description
+                        that includes relevant details.
+                    </p>
+                </div>
+                <div className={styles.card}>
+                    <div className={styles.iconContainer}>
+                        <img src={LightningSvg} alt="" className={styles.icon} />
+                    </div>
+                    <h6 className={styles.serviceCardTitle}>Service Title</h6>
+                    <p className={styles.serviceCardDescription}>
+                        This is the space to describe the service and explain how customers or
+                        clients can benefit from it. It’s an opportunity to add a short description
+                        that includes relevant details.
+                    </p>
+                </div>
+                <div className={styles.card}>
+                    <div className={styles.iconContainer}>
+                        <img src={GlobeSvg} alt="" className={styles.icon} />
+                    </div>
+                    <h6 className={styles.serviceCardTitle}>Service Title</h6>
+                    <p className={styles.serviceCardDescription}>
+                        This is the space to describe the service and explain how customers or
+                        clients can benefit from it. It’s an opportunity to add a short description
+                        that includes relevant details.
+                    </p>
+                </div>
+                <div className={styles.card}>
+                    <div className={styles.iconContainer}>
+                        <img src={LeafSvg} alt="" className={styles.icon} />
+                    </div>
+                    <h6 className={styles.serviceCardTitle}>Service Title</h6>
+                    <p className={styles.serviceCardDescription}>
```

Comment:
Using `<p>` elements in visual tools like Codux is problematic.

If the user drops a `<div>` or `<hr>` into `<p>` element, it will split it into two `<p>` elements: https://jsfiddle.net/1og7rsqm/

This is working according to the HTML spec, but it might be confusing for users. Because of that I think we should avoid using `<p>`.

Reply:
So, should I use div instead?

Reply:
Up to you. Depends on whether you think the issue I described is important.